src/Symfony/Component/PasswordHasher/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+CHANGELOG
+=========
+
+6.2
+---
+
+ * Hide sensitive information from stack traces with `SensitiveParameter` attribute
```

Comment:
Added the info to the changelog for discoverability.

Reply:
I think we need to be more precise about "Hide". It's "only" for back traces.